### PR TITLE
Information about "How to include video in docs" added.

### PR DESCRIPTION
**Pull request by troublemagnet**
I've added information about how one can add videos to the docs.
Please find the attached screenshot. 
how

Closes https://github.com/opendatakit/docs/issues/148.

### CODE REVIEW

File: contributing.rst [hunk 1]
```diff
@@ -1229,6 +1229,36 @@ Now, at the command line, from the root directory of the :file:`odk-docs` repo:
...
+
+To add a video in a document, you can do the following:
+
+
+.. code-block:: rst
+  
+  .. raw:: html
+
+  <video controls muted style="max-width:100%">
+    <source src="/{document-subdirectory}/{file}.*>
```
Comment (adammichaelwood) [inline contributing.rst]: unlike images, the file extension is required -- `.*` does not work.

File: contributing.rst [hunk 2]
```diff
@@ -1229,6 +1229,36 @@ Now, at the command line, from the root directory of the :file:`odk-docs` repo:
...
+
+**ADB or Android Debug Bridge** can be used to capture a screen recording from collect. This can be done by entering 
+
+.. code-block:: none
+
+  $ adb shell screenrecord /sdcard/example.mp4
+
+On pressing the enter key the video recording starts. Recording stops automatically after 3 minutes but since video length has to be less than a minute, to stop the recording in between simply press :command:`Ctrl+C`.
+
+The video file is saved in your Android device to a file at :file:`/sdcard/example.mp4` file.
```
Comment (adammichaelwood) [inline contributing.rst]: include the adb command for pulling the video locally

**Comment by adammichaelwood** [comment]
Great work. Two small fixes required.

I also encourage looking reviewing/editing your own writing thinking about extra words, clarity, etc. The prose could be cleaner, but rather than me doing a line-by-line comment thread I'd like you to think through it yourself for practice.

**Comment by adammichaelwood** [comment]
Thanks. Good work.

**Comment by troublemagnet** [comment]
@adammichaelwood Thank you sir. Hope to learn a lot while contributing to this project.

**Pull request merged**